### PR TITLE
Adds a skip to main content link for screen readers

### LINKED ISSUE #686: Add a skip-link
Skip-links are useful for those using keyboards to navigate the app, so that they can bypass repetitive blocks of content.

Learn more: https://a11yproject.com/posts/skip-nav-links/

### PR DESCRIPTION
**Pull request by WilHall**
This PR fixes #686:
 - Adds `tbds` skip link to main content container

### CODE REVIEW

**Comment by tysongach** [comment]
> @tysongach Does it make sense to hold on this for now and wait for the design system PR so we can use that facility?

Yeah, I think that makes sense. I’m sorry for not mentioning in #686 that we were planning to add a skip link component to the design system.

**Comment by WilHall** [comment]
No problem! Holding off for thoughtbot/design-system#129 - will revisit this after that is merged

**Comment by tysongach** [comment]
@WilHall [tbds v0.5.0](https://github.com/thoughtbot/design-system/releases/tag/v0.5.0) was just released, which includes the new [Skip Link component](https://github.com/thoughtbot/design-system/tree/master/src/skip-link).

**Comment by WilHall** [comment]
This PR has become quite small now!

File: lib/constable_web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -21,6 +21,7 @@
     <script src="<%= Application.fetch_env!(:constable, :shubox_script_url) %>"></script>
   </head>
   <body>
+  <a class="tbds-skip-link" href="#main">Skip to main content</a>
```
Comment (tysongach) [inline lib/constable_web/templates/layout/app.html.eex]:
```suggestion
    <a class="tbds-skip-link" href="#main">Skip to main content</a>
```

**Pull request merged**